doc/rose-rug-advanced-tutorials-suicide.html

```diff
@@ -260,6 +260,285 @@ <h3 class="alwayshidden">Note on suicide triggers (2)</h3>
         <li>triggering off a cleanup task in the suite</li>
       </ul>
     </div>
+
+    <div class="slide">
+          <h2 id="advanced-example">Advanced suicide triggers example</h2>
+      <p>The following example is one suite but with three possible 
+      scenarios:</p>
+
+      <ul>
+        <li>If a memeber of a family fails carry on.
+        If a specified task fails go to the end of that cycle and remove the 
```

Comment:
I think you actually want <dl> <dd> type markup as this reads like a set of bullets within bullets